docs/terms/data_docs_store.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Data Docs Store"
+---
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import TechnicalTag from '../term_tags/_tag.mdx';
+import SetupHeader from '/docs/images/universal_map/_um_setup_header.mdx'
+import CreateHeader from '/docs/images/universal_map/_um_create_header.mdx';
+import ValidateHeader from '/docs/images/universal_map/_um_validate_header.mdx';
+
+
+<UniversalMap setup='active' connect='active' create='active' validate='active'/> 
+
+## Overview
+
+### Definition
+
+A Data Docs Store is a connector to store and retrieve information pertaining to Human readable documentation generated from Great Expectations metadata detailing Expectations, Validation Results, etc.
+
+### Features and promises
+
+### Relationship to other objects
+
+Your Data Docs Store will be used behind the scenes by any Action that updates your Data Docs.  Great Expectations includes the `UpdateDataDocsAction` subclass of the `ValidationAction` class for this express purpose.  Including the `UpdateDataDocsAction` in the `action_list` of a Checkpoint will cause your Data Docs to be updated accordingly.
+
+## Use cases
+
+<SetupHeader/>
+
+When you configure your Data Docs, one of the items you will need to include is a `data_docs_store_backend` for each entry in the `data_docs_sites` key located in the `great_expectations.yml` file.  This will provide Great Expectations with the necessary information to access the Data Docs being rendered for each given site.  That process will generally be handled behind the scenes; after configuring your Data Docs sites to include a Data Docs Store, you will generally not have to directly interact with the Data Docs Store yourself.
+
+<CreateHeader/>
+
+When Profilers are run to create Expectations their results are made available through Data Docs.  Additionally, most Profiler workflows include a step where the Profiler's generated Expectation Suite is Validated against a Batch of data.  These processes use a Data Docs Store behind the scenes.
+
+<ValidateHeader/>
+
+When Checkpoints are run to Validate data, their results are generally updated into Data Docs.  Any Checkpoint that is configured to do so will use a Data Docs Store behind the scenes.  Including the `UpdateDataDocsAction` in the `action_list` of a Checkpoint will cause your Data Docs to be updated accordingly.  This update will only include the Validation Results from the Checkpoint itself and will not re-render all of your existing Data Docs to include Validation Results that were generated by a Checkpoint that did not include the `UpdateDataDocsAction` option.
```

Comment:
This last sentence slowed me down a bit -- i wonder if something like the following may be easier to parse: 

"This update will only include the Validation Results from the Checkpoint itself and will not re-render all of your existing Data Docs. If a Checkpoint does not include an `UpdateDataDocsAction`, the Validation Results will not be rendered into Data Docs." 

If the rest is

Reply:
I like that better as well.  Changed the content to match your suggestion.